src/TriggerCandidateMakerPlaneCoincidence.cpp

```diff
@@ -34,8 +40,8 @@ TriggerCandidateMakerPlaneCoincidence::operator()(const TriggerActivity& activit
 
       // add_window_to_record(m_current_window);
       // dump_window_record();
-      TLOG(1) << "Constructing trivial TC.";
-      TLOG(1) << "Activity count: " << m_activity_count;
+      TLOG_DEBUG(TLVL_DEBUG_LOW) << "[TCM:PC] Constructing trivial TC.";
```

Comment:
```suggestion
      TLOG_DEBUG(TLVL_DEBUG_LOW) << "[TCM:PC] Constructing a TC.";
```

Ditto w.g. "trivial"